test/evergreen.yml

```diff
@@ -2033,6 +2051,64 @@ tasks:
           test_config_filename: configs/reverse_split_stress.txt
           test_name: reverse_split
 
+# Cppsuite perf tests
+  - name: cppsuite-hs-cleanup-default-perf
```

Comment:
Could a template be used here to reduce the repeating nature?

Reply:
Looking around at other usages of templates I don't think they apply here, but I could wrong. Do you think there is a clean way to do this?

Reply:
It might not be possible, but take a look at workgen-test to see if that is an example you can leverage.

Reply:
It isn't possible as far as I can tell.

Reply:
Ok

Reply:
Is the reason because of the parameters? The example I provided used the task name to generate those values. I think we could do that here?

Reply:
Okay so upon further investigation this is technically possible if you use the mechanism workgen uses:
```
   FILENAME=`echo ${task_name}| cut -d'-' -f 3-`
```
But I personally don't want to use this as it then links, test name, config name pretty much everything back to the task name. Which would make customization a lot harder and less intuitive. Lets say I want to create a patch build, with a custom cppsuite config, I now have to go and edit my task name, or overwrite the existing config. Not the most intuitive experience.

I was assisted by @jiechenbo who agrees that the yml files do not have the functionality i'm looking for outside of doing what workgen does.

Reply:
This is something that would be far better generated. Happy to leave as is.